angular/src/docs/app/playground/playground.component.ts

```diff
@@ -14,6 +15,11 @@ import { Component, OnInit } from '@angular/core';
         <example-popover-showarrow></example-popover-showarrow>
       </div>
     </div>
+=======
+    <!-- Insert example here -->
+    <example-date-picker-default></example-date-picker-default>
+    <example-date-range-picker-default></example-date-range-picker-default>
+>>>>>>> feat(date-range-picker):create
```

Comment:
Looks like this merge needs to be cleaned up.

Reply:
Sure. Playground is always changing.